data/recipes.yaml

```diff
@@ -202,11 +202,10 @@
     - [10, solar panel]
   out:
     - [1, big furnace]
+## VEINS
```

Comment:
Putting this `## VEINS` comment here does not make sense, it is supposed to be a subsection of the `METAL` section.  Does the normalization script do this?

Reply:
That was me resolving the normalization change incorrectly.  I guess it's fixed now?